DAS/das/utils/logger.py

```diff
@@ -0,0 +1,235 @@
+from pathlib import Path
+import paho.mqtt.client as mqtt
+import paho.mqtt.publish as publish
+import csv
+import time
+import os
+import asyncio
+
+
+CsvConfig = {
+    "delimiter": ',',
+    "quotechar": '`',
+    "quoting": csv.QUOTE_ALL,
+    "skipinitialspace": True,
+    "fieldnames": ["time_delta", "mqtt_topic", "message"],
+}
+
+
+class Record:
```

Comment:
When you call the function it will be `logger.Record()`. It used to be `logger.Logger()` but I was getting confused when working on it. I'd be fine to change it to `Recorder` if others think its better

Reply:
`Record` isn't really a function though, it's a class. This becomes more obvious if you import it differently, so that it's not prefixed with the module name:
```python
from das.utils.logger import Record()
main_recorder = Record()
```
Considering the variable you're assigning the class to also contains `recorder`, I do kinda feel like the class should be called that and the assignment would make more sense. Happy to hear more opinions though!

Reply:
Totally agree, added a start function. It makes it significantly easier to work with!